README.md

```diff
@@ -132,6 +132,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [Farfetch](https://www.farfetch.com) [:rocket:](https://www.farfetch.com/pt/careers) | Global technology platform for luxury fashion. | `Braga` `Lisboa`<br>`Porto` |
 | [Hostelworld](https://www.hostelworld.com/) [:rocket:](http://www.hostelworldgroup.com/careers) | World's leading hostel-focused online booking platform. | `Porto` |
 | [Jumia](https://group.jumia.com) [:rocket:](https://group.jumia.com/careers) | Online Shopping for Electronics, Phones & Fashion in Africa. | `Porto` |
+| [La Redoute](https://www.laredoute-corporate.com/en/talents-en/) [:rocket:](https://www.linkedin.com/company/la-redoute/jobs/) | Home & Fashion online. | `Leiria` |
```

Comment:
Hi @DanielOlival, just want to make sure the first link is OK.
Usually it points to the main page, not to a specific one.
Just let me know if this is the right one or else please update it.

Thanks!

Reply:
Hello @marmelo  , i've updated the site to the main page

Thanks!